packages/node_modules/@ciscospark/plugin-conversation/src/share-activity.js

```diff
@@ -233,9 +233,9 @@ const ShareActivity = SparkPlugin.extend({
       verb: `share`,
       object: {
         objectType: `content`,
-        displayName: this.displayName,
-        content: this.content,
-        mentions: this.mentions,
+        displayName: this.object && this.object.displayName ? this.object.displayName : ``,
```

Comment:
I'm suggesting that `ShareActivity#object` should never be defined. When we started down the `ShareActivity` path, the intent was to have all of the client-settable properties of the activity be top-level properties of the `ShareActivity` instance.